src/pages/Lesson8.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC } from 'react';
+import AppWrapper from '../components/AppWrapper';
+import CompareCode from '../components/CompareCode';
+import NavButtons from '../components/NavButtons';
+import { HeaderSections } from '../types/globalTypes';
+
+const Lesson8: FC = () => {
+  const code1 = `
+    #include <iostream>
+
+    Void addFiveNormal(int x) 
+    {
+        X += 5;
+    }
+
+    Void addFivePointer(int* x) 
+    {
+        *X += 5;
+    }
+
+    Void addFiveReference(int &x) 
+    {
+        X += 5;
+    }
+
+    Int main() 
+    {
+        Int first = 0;
+        Int second = 0;
+        Int third = 0;
+        addFiveNormal(first);
+        addFivePointer(&second);
+        addFiveReference(third);
+        std::cout << first << endl;
+        std::cout << second << endl;
+        std::cout << third << endl;
+    }
+
+    `;
+
+  const code2 = `
+    Our Output:
+    0
+    5
+    5
+    `;
+
+  return (
+    <>
+      <AppWrapper section={HeaderSections.LESSON_8}>
+        <div className="page-wrapper">
+          <h1>References vs. Pointers</h1>
+          <br />
+          <p>
+            Now let&apos;s take a look at one way that pointers can be useful!
+          </p>
+          <div className="diagram">
+            <CompareCode
```

Comment:
Let's add captions here instead-
"Code" on the left
"Output" on the right

Also, can you try making the right side centered like the Figma?
![image](https://github.com/uclaacm/parcel-pointers/assets/78634083/97b066c9-c1f9-463b-a9c4-6c2033aef303)
![image](https://github.com/uclaacm/parcel-pointers/assets/78634083/6bf18c78-3155-4169-9717-48068208d85d)
You may need to modify the component itself, let me know if this is does not work, but the component is only used in lessons 5 and 6 as well, so I don't foresee any issues.